packages/frontend/src/hooks/echarts/useEcharts.ts

```diff
@@ -66,6 +66,46 @@ const getAxisTypeFromField = (item?: Field): string => {
     }
 };
 
+export const getAxisDefaultMinValue = ({
+    min,
+    max,
+}: {
+    min: any;
+    max: any;
+}) => {
+    if (
+        isNaN(parseInt(min)) ||
```

Comment:
why not parse float?

Reply:
It's only purpose was to handle edge cases with strings. E.g `isNan("")` returns true :/ 
I can remove completely the parseInt and just check for empty strings.